docs/cypress-cloud.md

```diff
@@ -0,0 +1,4 @@
+Could you navigate to this URL in interactive mode (browserPort = 7333)
```

Comment:
Intentional to check in these docs files?

Reply:
the plan is to eventually have Cline debug Cypress cloud failures and automate tests so was planning on having those prompts here

Reply:
It feels too Roo-specific to me now that we have a bunch of other people in this code